Externals/SlippiRustExtensions/jukebox/src/errors.rs

```diff
@@ -0,0 +1,37 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum JukeboxError {
+    #[error("{0}")]
+    GenericIOError(#[from] std::io::Error),
```

Comment:
Stylistic bit but: you could drop the `Error` from the variant types here to reduce noise, e.g:

`JukeboxError::GenericIOError` vs `JukeboxError::GenericIO` (or just `JukeboxError::IO`).